symbiflow.py

```diff
@@ -385,8 +385,17 @@ def get_step_runtime(step, logfile, position):
         fasm_log = os.path.join(self.out_dir, 'fasm.log')
 
         log['pack'] = get_step_runtime('Packing', pack_log, 3)
+        log['overhead'] = get_step_runtime(
+            'The entire flow of VPR', pack_log, 6
```

Comment:
This run-time actually includes also the time required for the actual packing. It would be better to take, for instance the following run-tmes:
```
# Clean circuit took 17.81 seconds (max_rss 1115.7 MiB, delta_rss +943.9 MiB) 
...
# Load Timing Constraints took 3.03 seconds (max_rss 1118.1 MiB, delta_rss +0.0 MiB) 
...
# Create Device took 6.76 seconds (max_rss 3472.8 MiB, delta_rss +2354.8 MiB)

```

Given that there are several of these, it would be good maybe to have a function that adds all these runtimes for each P&R step.